### PR TITLE
Push Revamp

### PR DESCRIPTION
**Pull request by EmilyNecciai**
# Pull Request/Issue Resolution

**Description of Change:**
> I'm changing..... (could be a link, a new image, a new section, etc.)... because...

Closes #**ISSUE_NUMBER_HERE**

### Is this change associated with a Braze feature/product release?
- [ ] Yes (__Insert Feature Release Date Here__)
- [ ] No

> If yes, please note the date of the feature release.

---
---

<details>
<summary>PR Checklist</summary>
- [ ] Check that all links work!
- [ ] Ensure you have completed [our CLA](https://www.braze.com/docs/cla/).
- [ ] Tag @EmilyNecciai as a reviewer when the your work is _done and ready to be reviewed for merge_.
- [ ] Tag others as Reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `config/nginx.conf.erb`.

</details>

<details>
<summary>Writing & Style Quick Tips</summary>

- Use the formatting shown on the [Styling Test Page](https://github.com/Appboy/braze-docs/blob/develop/_docs/_home/styling_test_page.md) to create unique templates and vehicles for your information.
- Use Action verbs!
- Capitalize proper nouns, unique Braze tools and features, custom Braze channels, and all Header titles.
- Redirect when necessary!
- Always speak to the Braze customer - refer to them as “you”, whether they are an Engineer/Developer, a Marketer, or an Admin.
- On Voice: You should aim to be a Friendly Professor. Imagine, if you will, the smartest, but also kindest professor you had. You know, the one who showed you your potential and tested you, but also went slowly in class so you could take great notes and learn some things? Don't be too peppy, but don't be formal. You’re not selling anything - you’re instructing and informing and enabling.

_Consult the [Docs Text Formatting Guide](https://github.com/Appboy/braze-docs/wiki/Special-Formatting) and the [Docs Writing Style Guide & Best Practices](https://github.com/Appboy/braze-docs/wiki/Writing-Style-Guide-&-Best-Practices) for more details._

</details>

<details>
<summary>ATTENTION: For PR Reviewers</summary>
- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Preview all changes in the linked Heroku environment (click `View deployment` button below, then "Docs". A `502` error just means you should refresh until you see the Docs Home page.
</details>

---
---

### CODE REVIEW

File: _docs/_user_guide/data_and_analytics/user_data_collection/user_import.md [hunk 1]
```diff
@@ -42,23 +42,23 @@ Note that setting `language` and/or `country` on a user via CSV import or API wi
...
-| `phone` | String | A telephone number as indicated by your users (e.g. `541 754 3010`). | No |
-| `email_subscribe` | String | Available values are `opted_in` (explicitly registered to receive email messages), `unsubscribed` (explicitly opted out of email messages), and `subscribed` (neither opted in nor out). | No |
-| `push_subscribe` | String | Available values are `opted_in` (explicitly registered to receive push messages), `unsubscribed` (explicitly opted out of push messages), and `subscribed` (neither opted in nor out). | No |
-| `time_zone` | String | Time zone must be passed to Braze in the same format as the IANA Time Zone Database (e.g. `America/New_York` or `Eastern Time (US & Canada)`).  | No |
-| `date_of_first_session` <br><br> `date_of_last_session`| String | May be passed in one of the following ISO8601 formats: <br> - "YYYY-MM-DD" <br> - "YYYY-MM-DDTHH:MM:SS+00:00" <br> - "YYYY-MM-DDTHH:MM:SSZ" <br> - "YYYY-MM-DDTHH:MM:SS" (e.g. `2019-11-20T18:38:57`) | No |
-| `image_url` | String | A URL of an image  | No |
+| <nobr><code>external_id</code></nobr> | String | A unique user identifier for your customer. | Yes, see following note |
```
Comment (EmilyNecciai) [inline _docs/_user_guide/data_and_analytics/user_data_collection/user_import.md]: @KellieHawks prevents line breaking but gives a funky format. Probably need CSS to make consistent.

File: _docs/_user_guide/message_building_by_channel/push/about.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+description:
+channel: push
+---
+# What are Push Messages?
+
+Push notifications are wonderful for time-sensitive calls to action, as well as re-engaging users who haven't come into the app in a while.
+
+Successful push campaigns drive the user directly to content and demonstrate the value of your app.
+
+_To see more examples of push notifications, check out our [Client Integration Gallery][8]._
```
Comment (EmilyNecciai) [inline _docs/_user_guide/message_building_by_channel/push/about.md]: Now the Customer Examples Library: https://www.braze.com/docs/help/best_practices/client_integration_gallery/

File: _docs/_user_guide/message_building_by_channel/push/android/advanced_campaign_settings.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+---
+nav_title: Advanced Push Campaign Settings
+---
+Hi
```
Comment (EmilyNecciai) [inline _docs/_user_guide/message_building_by_channel/push/android/advanced_campaign_settings.md]: This is this content: https://www.braze.com/docs/developer_guide/platform_integration_guides/android/push_notifications/advanced_settings/

Comment (EmilyNecciai) [reply _docs/_user_guide/message_building_by_channel/push/android/advanced_campaign_settings.md]: either 2 different pages for dev and for marketer 
or
move all content to user guide

(probably the 1st)

File: _docs/_user_guide/message_building_by_channel/push/android/notification_ids.md [hunk 4]
```diff
@@ -1,7 +1,5 @@
 ---
 nav_title: Notification IDs
```
Comment (EmilyNecciai) [inline _docs/_user_guide/message_building_by_channel/push/android/notification_ids.md]: @KellieHawks Most of these pages need metadata or need updated metadata

File: _docs/_user_guide/message_building_by_channel/push/android/push-enabled.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+---
+nav_title: Push-Enabled Users
+
+page_type: reference
+platform: android
+channel: push
+---
+
+# What makes an Android user "Push-Enabled"?
```
Comment (EmilyNecciai) [inline _docs/_user_guide/message_building_by_channel/push/android/push-enabled.md]: This really needs to be elaborated on - 

1. what does Push Enabled mean for Android
2. Technical requirements for being push enabled
3. how your users can become push enabled. 
4. Sample campaign/canvas?

File: _docs/_user_guide/message_building_by_channel/push/create_push_primer.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+For a user to qualify to receive your Push Messages, they must enable Push at the app-level _and_ the device-level. Please note that these levels translate differently for iOS and Android. You can learn more about them here:
+- [Android Push Enabled]({{ site.baseurl }}/)
+- [iOS Push Enabled]
+
+## Step 1: Select Your Channel of Choice
+
+From the Messaging Page, click "Create Campaign."
+
+
+## Step 2:
```
Comment (EmilyNecciai) [inline _docs/_user_guide/message_building_by_channel/push/create_push_primer.md]: @KellieHawks Talk to a CSM like TG for info on this!

File: _docs/_user_guide/message_building_by_channel/push/creating_a_push_message.md [hunk 7]
```diff
@@ -1,30 +1,17 @@
...
-|[4.5.4][9] Push Notifications must not be required for the app to function, and should not be used for advertising, promotions, or direct marketing purposes or to send sensitive personal or confidential information.|
-|[3.2.2][10] (i) Creating an interface for displaying third party apps, extensions, or plug-ins similar to the App Store or as a general-interest collection. (ii) Monetizing built-in capabilities provided by the hardware or operating system, such as Push Notifications, the camera, or the gyroscope; or Apple services, such as Apple Music access or iCloud storage.|
-
-|Google Play Store Policy|
-|---|
-|[We don’t allow apps or ads that mimic or interfere with system functionality, such as notifications or warnings.][11] System level notifications may only be used for an app’s integral features, such as an airline app that notifies users of special deals, or a game that notifies users of in-game promotions.|
-
-
-
+Successful push campaigns drive the user directly to content and demonstrate the value of your app.
```
Comment (EmilyNecciai) [inline _docs/_user_guide/message_building_by_channel/push/creating_a_push_message.md]: General walkthrough of creating a push campaign
- copy from the campaign page but make it for push. 
- add a few notes about character truncation

Comment (EmilyNecciai) [reply _docs/_user_guide/message_building_by_channel/push/creating_a_push_message.md]: Partner with AK for a video

File: _docs/_user_guide/message_building_by_channel/push/create_push_primer.md (hunk 6 above)
Comment (KellieHawks) [reply _docs/_user_guide/message_building_by_channel/push/create_push_primer.md]: Reached out to Taylor for additional steps.

**Comment by KellieHawks** [comment]
@cla-bot check

**Comment by KellieHawks** [comment]
@cla-bot check

**Comment by KellieHawks** [comment]
@dwillislc, 
Ready to get this up with today's deploy. Please approve sometime today. Thank you!

**Comment by dwillislc** [comment]
this looks great. merge it!

**Pull request merged**